docs/source/ruletypes.rst

```diff
@@ -510,6 +524,85 @@ This value is added in back of the event. For example,
 
 ``kibana4_end_timedelta: minutes: 2``
 
+generate_kibana_discover_url
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``generate_kibana_discover_url``: Enables the generation of the ``kibana_discover_url`` variable for the Kibana Discover application.
+This setting requires the following settings are also configured:
+
+- ``kibana_discover_app_url``
+- ``kibana_discover_version``
+- ``kibana_discover_index_pattern_id``
+
+``generate_kibana_discover_url: true``
+
+kibana_discover_app_url
+^^^^^^^^^^^^^^^^^^^^^^^
+
+``kibana_discover_app_url``: The url of the Kibana Discover application used to generate the ``kibana_discover_url`` variable.
+This value can use `$VAR` and `${VAR}` references to expand environment variables.
+
+``kibana_discover_app_url: http://kibana:5601/#/discover``
+
+kibana_discover_version
+^^^^^^^^^^^^^^^^^^^^^^^
+
+``kibana_discover_version``: Specifies the version of the Kibana Discover application.
+
+The currently supported versions of Kibana Discover are: 
+
+- `5.6`
+- `6.0`, `6.1`, `6.2`, `6.3`, `6.4`, `6.5`, `6.6`, `6.7`, `6.8`
+- `7.0`, `7.1`, `7.2`, `7.3`
+
+``kibana_discover_version: '7.3'``
+
```

Comment:
I decided to go with an explicit version for a couple reasons. First, the Discover state, while it's been relatively consistent, isn't a well defined public contract. There will be changes. They could happen in a minor release with the addition of new functionality.

The second reason is that there are tones of issues asking if the Kibana4 dashboard apps supports the newer versions. This hopefully clarifies and addresses what is supported.